clicache/integration-test/CacheHelperN.cs

```diff
@@ -2143,14 +2147,23 @@ static int ExecuteGfsh(string command)
           }
         }
 
-        //process.CancelOutputRead();
-        //process.CancelOutputRead();
-
         Util.Log("ExecuteGfsh: Exited {0}", process.Id);
         return process.ExitCode;
       }
     }
 
+    public static void CloseAndIgnore(StreamReader streamRead)
+    {
+      try
+      {
+        streamRead.Close();
+      } 
+      catch (Exception)
+      {
+        // ignored
```

Comment:
I take it this was the site of the `AppDomainUnloadedException`?

Reply:
No this is to ignore exceptions on closing file handles to sub processes rather than failing the test.